lib/index.js

```diff
@@ -76,15 +76,15 @@ function printResults(results, c) {
     }).join("\n");
 }
 
-function searchVOD(vodID, c) {
+function searchVOD(vodID, colorize, clientId) {
```

Comment:
I had problems with it passing the colorize method instead when I named the colorize param colorize. Even though it in theory should not affect it. I'll try the pull locally and if it works merge it :)